packages/design-tokens/css/default-theme.css

```diff
@@ -1,302 +1,1696 @@
+  --kz-var-color-coral-200-rgb: 252, 207, 212;
   --kz-var-color-coral-200-rgb-params: 252, 207, 212;
+  --kz-var-color-coral-200-rgb-id: --kz-var-color-coral-200-rgb;
+  --kz-var-color-coral-200-id: --kz-var-color-coral-200;
   --kz-var-color-coral-300: #fcaeb6;
+  --kz-var-color-coral-300-rgb: 252, 174, 182;
   --kz-var-color-coral-300-rgb-params: 252, 174, 182;
+  --kz-var-color-coral-300-rgb-id: --kz-var-color-coral-300-rgb;
+  --kz-var-color-coral-300-id: --kz-var-color-coral-300;
   --kz-var-color-coral-400: #fc7c89;
+  --kz-var-color-coral-400-rgb: 252, 124, 137;
   --kz-var-color-coral-400-rgb-params: 252, 124, 137;
+  --kz-var-color-coral-400-rgb-id: --kz-var-color-coral-400-rgb;
+  --kz-var-color-coral-400-id: --kz-var-color-coral-400;
   --kz-var-color-coral-500: #e22b3e;
+  --kz-var-color-coral-500-rgb: 226, 43, 62;
   --kz-var-color-coral-500-rgb-params: 226, 43, 62;
+  --kz-var-color-coral-500-rgb-id: --kz-var-color-coral-500-rgb;
+  --kz-var-color-coral-500-id: --kz-var-color-coral-500;
   --kz-var-color-coral-600: #b02130;
+  --kz-var-color-coral-600-rgb: 176, 33, 48;
   --kz-var-color-coral-600-rgb-params: 176, 33, 48;
+  --kz-var-color-coral-600-rgb-id: --kz-var-color-coral-600-rgb;
+  --kz-var-color-coral-600-id: --kz-var-color-coral-600;
   --kz-var-color-coral-700: #6c1e20;
+  --kz-var-color-coral-700-rgb: 108, 30, 32;
   --kz-var-color-coral-700-rgb-params: 108, 30, 32;
+  --kz-var-color-coral-700-rgb-id: --kz-var-color-coral-700-rgb;
+  --kz-var-color-coral-700-id: --kz-var-color-coral-700;
+  --kz-var-color-red-100: #fcedef;
+  --kz-var-color-red-100-rgb: 252, 237, 239;
+  --kz-var-color-red-100-rgb-params: 252, 237, 239;
+  --kz-var-color-red-100-rgb-id: --kz-var-color-red-100-rgb;
+  --kz-var-color-red-100-id: --kz-var-color-red-100;
+  --kz-var-color-red-200: #fccfd4;
+  --kz-var-color-red-200-rgb: 252, 207, 212;
+  --kz-var-color-red-200-rgb-params: 252, 207, 212;
+  --kz-var-color-red-200-rgb-id: --kz-var-color-red-200-rgb;
+  --kz-var-color-red-200-id: --kz-var-color-red-200;
+  --kz-var-color-red-300: #fcaeb6;
+  --kz-var-color-red-300-rgb: 252, 174, 182;
+  --kz-var-color-red-300-rgb-params: 252, 174, 182;
+  --kz-var-color-red-300-rgb-id: --kz-var-color-red-300-rgb;
+  --kz-var-color-red-300-id: --kz-var-color-red-300;
+  --kz-var-color-red-400: #fc7c89;
+  --kz-var-color-red-400-rgb: 252, 124, 137;
+  --kz-var-color-red-400-rgb-params: 252, 124, 137;
+  --kz-var-color-red-400-rgb-id: --kz-var-color-red-400-rgb;
+  --kz-var-color-red-400-id: --kz-var-color-red-400;
+  --kz-var-color-red-500: #e22b3e;
+  --kz-var-color-red-500-rgb: 226, 43, 62;
+  --kz-var-color-red-500-rgb-params: 226, 43, 62;
+  --kz-var-color-red-500-rgb-id: --kz-var-color-red-500-rgb;
+  --kz-var-color-red-500-id: --kz-var-color-red-500;
+  --kz-var-color-red-600: #b02130;
+  --kz-var-color-red-600-rgb: 176, 33, 48;
+  --kz-var-color-red-600-rgb-params: 176, 33, 48;
+  --kz-var-color-red-600-rgb-id: --kz-var-color-red-600-rgb;
```

Comment:
Why do we need `-rgb-id`?

Reply:
In case you want to do a decision token with `rgba`
![image](https://user-images.githubusercontent.com/7651503/125888034-d8d0fc64-3896-4768-9ccb-38ee16a1049f.png)

Reply:
Ah 🤦🏻 My bad. Yes, that's right.